test/lib/view.js

```diff
@@ -0,0 +1,492 @@
+const t = require('tap')
+const mr = require('npm-registry-mock')
```

Comment:
just a tip, I have been finding it way simpler to just mock the lib that reaches for the registry in the case of unit tests (in this case `pacote`) then setting up a complete registry mock